test/uk/gov/hmrc/serviceconfigs/persistence/AlertEnvironmentHandlerRepositorySpec.scala

```diff
@@ -72,7 +72,7 @@ class AlertEnvironmentHandlerRepositorySpec
       eventually {
         repository.insert(Seq(alertEnvironmentHandler))
         repository.findAll().futureValue must contain(alertEnvironmentHandler)
-        repository.deleteAll
+        repository.deleteAll.futureValue
```

Comment:
does the `insert` also need a `.futureValue`?
It might be better to put this into a for-comprehension which will guarantee that they are all being sequenced - and just call `.futureValue` once at the end

Reply:
Yeah, good catch, I've removed the `eventually` (which I think was actually hiding the flakiness of these tests but resulting in the suite taking much longer to run than it needed)